src/components/CalendarMonth.jsx

```diff
@@ -25,6 +26,7 @@ const propTypes = forbidExtraProps({
   enableOutsideDays: PropTypes.bool,
   modifiers: PropTypes.object,
   orientation: ScrollableOrientationShape,
+  daySize: nonNegativeInteger,
```

Comment:
in fact nonNegativeInteger isn't thunkified, so this is correct as-is https://unpkg.com/airbnb-prop-types@2.8.0/src/nonNegativeInteger.js